snaps/reference/cli/subcommands.md

```diff
@@ -29,6 +29,32 @@ Builds a Snap from source.
 
 `b` is an alias for `build`.
 
+#### `analyze`
+
+<Tabs>
+<TabItem value="Syntax">
+
+```bash
+yarn mm-snap build --analyze <BOOLEAN>
```

Comment:
It technically takes a boolean value, but since it's disabled by default, it's easier to think of the flag as enabling the bundle analyser. So just specifying this will enable it:

```suggestion
yarn mm-snap build --analyze
```